cfy_manager/components/restservice/restservice.py

```diff
@@ -539,5 +540,6 @@ def upgrade(self):
         logger.notice('Upgrading Rest Service...')
         self._deploy_restservice_files()
         run_script_on_manager_venv('/opt/manager/scripts/load_permissions.py')
+        run_script_on_manager_venv('/opt/manager/scripts/create_system_filters.py') # NOQA
```

Comment:
Maybe wrap the line rather than noqaing it, since noqa skips other checks as well as just line length.

Reply:
Done